imagedecoder.raw/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="imagedecoder.raw"
-  version="2.0.1"
+  version="2.0.2"
```

Comment:
I don't see the need for a version bump.

Reply:
I can remove found also another very big bug inside (fix come soon). Like on the other generate it a shared on depends, where is after zip created no more available and the addon can never work by the repo way!